src/app/body/board/sprint-details/sprint-details.component.html

```diff
@@ -49,23 +49,27 @@ <h5 class="pl-2" style="font-size: 18px;">Sprint No: {{currentSprintName}}</h5>
         </div>
         <div class="card-body">
             <div class="row py-2">
-                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">Start Date:
+                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 15px;">Start Date:
                 </h6>
-                <h6 class="card-text col-md-10 col-8" id="sprintStartDate" style="font-size: 12px;">
+                <h6 class="card-text col-md-10 col-8" id="sprintStartDate" style="font-size: 13px;">
                     {{sprintData.StartDate}}</h6>
             </div>
             <div class="row py-2">
-                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">End Date: </h6>
-                <h6 class="card-text col-md-10 col-8" id="sprintEndDate" style="font-size: 12px;">{{sprintData.EndDate}}
+                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 15px;">End Date: </h6>
+                <h6 class="card-text col-md-10 col-8" id="sprintEndDate" style="font-size: 13px;">{{sprintData.EndDate}}
                 </h6>
             </div>
             <div class="row py-2">
-                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">Status: </h6>
-                <h6 class="card-text col-md-10 col-8" id="sprintStatus" style="font-size: 12px;">{{sprintData.Status}}</h6>
+                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 15px;">Status: </h6>
+                <h6 class="card-text col-md-10 col-8" id="sprintStatus" style="font-size: 13px;">{{sprintData.Status}}</h6>
             </div>
             <div class="row py-2">
-                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">Number of task: </h6>
-                <h6 class="card-text col-md-10 col-8" id="sprintStatus" style="font-size: 12px;">{{sprintData.TotalNumberOfTask}}</h6>
+                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 15px;">Total Tasks: </h6>
+                <h6 class="card-text col-md-10 col-8" id="sprintStatus" style="font-size: 13px;">{{sprintData.TotalNumberOfTask}}</h6>
+            </div>
+            <div class="row py-2">
+                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 15px;">Completed Tasks: </h6>
+                <h6 class="card-text col-md-10 col-8" id="sprintStatus" style="font-size: 13px;">{{sprintData.TotalCompletedTask}}</h6>
```

Comment:
Put these in first giving it a column, so that it consumes the space of the right